src/styles/base16/3024.css

```diff
@@ -34,16 +34,6 @@ base0E  #a16a94  Keywords, Storage, Selector, Markup Italic, Diff Changed
 base0F  #cdab53  Deprecated, Opening/Closing Embedded Language Tags, e.g. <?php ?>
 */
 
-pre code.hljs {
```

Comment:
I'm not sure if the base16 files go thru our internal build process, they have their own external build process, but I'll double check.